aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/BaseHandlerStd.java

```diff
@@ -62,6 +63,15 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
                     InvalidCustomDbEngineVersionStateException.class)
             .build();
 
+    protected static final ErrorRuleSet UPDATE_DELETE_CUSTOM_DB_ENGINE_VERSION_ERROR_RULE_SET = ErrorRuleSet
```

Comment:
The name is hard to interpret. Let's try to reflect its actual effect (interprets AccessDenied as NotFound) rather than where it is applied.